fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs

```diff
@@ -0,0 +1,271 @@
+#define FIXED // FIRST ADMIN
+#if NEVER
+#elif FIXED
+using Microsoft.AspNetCore.RateLimiting;
+using System.Threading.RateLimiting;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+static string GetTicks() => DateTime.Now.Ticks.ToString().Substring(14);
```

Comment:
@fiyazbinhasan can you think of a better way to get the last 4 digits of `Ticks`?

Reply:
`(DateTime.Now.Ticks & 0x1111).ToString("0000");`